tests/optimisation_test/optimisation_manager.py

```diff
@@ -39,21 +38,60 @@ def is_recipe_template_available(self, _template_path):
 
     #optimization_part
     
-    def generate_bound_from_config():
-        pass
+    def _generate_bound_from_config(self):
+        constraint = {}
+        for key in self._config_dict['experiment']['components']:
+            constraint[key] = (self._config_dict['experiment']['components'][key]['lower_bound'],
+                               self._config_dict['experiment']['components'][key]['upper_bound'])
+        return constraint
+
+    def _generate_model(self):
+        """
+        Generate optimization model from config file based on the 'optimizer' part.
+        If the model requires constraint information, please pass it via kwargs.
+        """
+        module = importlib.import_module(self._config_dict['optimizer']['module'])
+        optimizer_class = getattr(module, self._config_dict['optimizer']['class'])
+        return optimizer_class(**self._config_dict['optimizer']['hyperparameters'], **kwargs)
```

Comment:
remove **kwargs as these should be defined inside the config dictionary as parameters or args